docs/src/languages/javascript.md

```diff
@@ -85,3 +85,21 @@ You can configure ESLint's `nodePath` setting (requires Zed `0.127.0`):
   }
 }
 ```
+
+#### Configure ESLint's `problems.shortenToSingleLine`:
+
+You can configure ESLint's `problems.shortenToSingleLine` setting (requires Zed `0.129.2`):
```

Comment:
I didn't see anything in the contributing guide for how to handle versions in docs, so I've just put in what I assume the next release version will be (happy to adjust).

Reply:
In this case it would be `0.130.x` if we merge before Wednesday evening. Releases go out every Wednesday, and preview becomes stable, and preview gets a new minor version.